modules/metrics/src/main/java/org/apache/ignite/internal/metrics/SimpleMovingAverage.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.metrics;
+
+import java.util.concurrent.ConcurrentLinkedDeque;
+import java.util.function.DoubleFunction;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * The metric calculates the average value for the last several operations.
+ */
+public class SimpleMovingAverage extends AbstractMetric implements DoubleMetric {
+    /** Default window size. */
+    public static final int DFLT_ITEMS = 100;
+
+    /** Size. */
+    private final int items;
+
+    /** Function to format a readable value. */
+    private final DoubleFunction<String> stringFormatter;
+
+    /** Elements. */
+    ConcurrentLinkedDeque<Double> queue = new ConcurrentLinkedDeque<>();
+
+    /**
+     * The constructor.
+     *
+     * @param name Name.
+     * @param desc Description.
+     * @param stringFormatter String formatter to get a readable value.
+     */
+    public SimpleMovingAverage(String name, @Nullable String desc, @Nullable DoubleFunction<String> stringFormatter) {
```

Comment:
As I understand we agreed to remove formatter for now?

Reply:
Fixed.